MyPerfectOnboardingApplication/Tests/MyPerfectOnboarding.App.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("MyPerfectOnboarding.App.Tests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("MyPerfectOnboarding.App.Tests")]
```

Comment:
```suggestion
[assembly: AssemblyProduct("MyPerfectOnboarding.Application.Tests")]
```
https://github.com/HanaNavratilova/kentico-onboarding-cs/pull/4#discussion_r261535994 💩